localstack-core/localstack/utils/bootstrap.py

```diff
@@ -1358,10 +1358,11 @@ def in_ci():
     return False
 
 
-def is_api_key_configured() -> bool:
+def is_auth_token_configured() -> bool:
```

Comment:
i see you already made the changes in ext, but feel that leaving `is_api_key_configured` just in case would be useful. by creating code paths like this we leave a little remnant in there but at least decouple the dependency. anyway, just a nit

```python
def is_api_key_configured():
    return is_auth_token_configured()
```